sections/product-page.liquid

```diff
@@ -1,67 +1,11 @@
-{{ "product-page.css" | asset_url | stylesheet_tag }}
-{{ "carousels.css" | asset_url | stylesheet_tag }}
-
 {% if canonical_url contains "/1?" or canonical_url contains "/1#" %}
   {% include "product-page-preview" %}
 {% else %}
-  <div class="product-gallery">
-    <div class="product-gallery__preview carousel carousel--snap-center">
-      <div class="product-gallery__preview-inner carousel__inner infinite-scroll">
-        {% for image in product.images %}
-          <div class="product-gallery__preview-item carousel__item">
-            {{ image.url | image_tag:
-              class: 'lazyload',
-              widths: '600, 700, 800',
-              sizes: '(max-width: 576px) 600px, (max-width: 876px) 700px, 800px',
-            }}
-          </div>
-        {% endfor %}
-      </div>
-      {% if product.images.size > 1 %}
-        <div class="carousel__controls">
-          <div class="carousel__control prev">
-            <i class="fa-light fa-arrow-left"></i>
-          </div>
-          <div class="carousel__control next">
-            <i class="fa-light fa-arrow-right"></i>
-          </div>
-        </div>
-      {% endif %}
-    </div>
-    {% if product.images.size > 1 %}
-    <div class="product-gallery__items carousel carousel--snap-center">
-      <div class="carousel__inner">
-        {% for image in product.images %}
-          {% assign focal = image.coordinates %}
-          <div class="product-gallery__item-container carousel__item" data-index="{{ forloop.index0 }}">
-            {{ image.url | image_tag:
-              loading: 'lazy',
-              focal: focal,
-              widths: '150'
-            }}
-          </div>
-        {% endfor %}
-      </div>
-    </div>
-    {% endif %}
-  </div>
-  <div class="product-details">
-    <div class="product-info">
-      <h1 class="product-info__title">{{ product.name }}</h1>
-      {% if product.variants.size > 1 %}
-        <p class="product-info__label">{{ section.settings.variation_label }}</p>
-      {% endif %}
-      {{ product_button }}
-    </div>
-    {% if product.description != blank %}
-      <div class="product-description">
-        <p class="product-description__label">{{ section.settings.description_label }}</p>
-        <div class="product-description__content-container bq-content rx-content">
-          {{ product.description }}
-        </div>
-      </div>
-    {% endif %}
-  </div>
+  {% if enabled_features contains "next_product_page" %}
+    {% include "product-page-next" %}
```

Comment:
Since the legacy page has the suffix `-legacy`, maybe we don't need a suffix `-next`?

```suggestion
    {% include "product-page" %}
```